.github/workflows/pr.yaml

```diff
@@ -18,6 +20,7 @@ jobs:
       - wheel-build-arm64
       - wheel-test-amd64
       - wheel-test-arm64
+      - wheel-publish-pynvjitlink
```

Comment:
This applies a dependency to a job that is skipped unless a tag is pushed, and so will only succeed on tag pushes, I think.

Reply:
this maybe resolved by https://github.com/rapidsai/pynvjitlink/pull/17/commits/a854b453eee6989336a139ac538b9e56510b72bd

Reply:
This appears to be resolved if we're ok with the above.